src/Stripe.net/Constants/Events.cs

```diff
@@ -568,6 +568,21 @@ public static class Events
         /// </summary>
         public const string PlanUpdated = "plan.updated";
 
+        /// <summary>
```

Comment:
Those are not public yet but should be. I'm adding those because it avoids an extra PR once we fix the issue internally.